src/mongocrypt.h

```diff
@@ -25,14 +25,18 @@
 
 #include "mongocrypt-export.h"
 #include "mongocrypt-compat.h"
-#include "mongocrypt-config.h"
 
-/**
- * @def MONGOCRYPT_VERSION
- * The version string describing libmongocrypt.
- * Has the form x.y.z-<pre>+<date>+git<sha>.
- */
-#define MONGOCRYPT_VERSION "@MONGOCRYPT_BUILD_VERSION@"
+/* clang-format off */
```

Comment:
Suggest removing the disabling of `clang-format` within this code block.

Side note: is the clang-format option `IndentPPDirectives` worth considering?

`IndentPPDirectives: BeforeHash`:
```c
#ifndef __has_include
   #include "mongocrypt-config.h"
#else
   #if __has_include("mongocrypt-config.h")
      #include "mongocrypt-config.h"
   #else
      #error \
         "No 'mongocrypt-config.h' header is available. This must be generated in order to use libmongocrypt"
   #endif
#endif
```

`IndentPPDirectives: AfterHash` (my preference):
```c
#ifndef __has_include
#   include "mongocrypt-config.h"
#else
#   if __has_include("mongocrypt-config.h")
#      include "mongocrypt-config.h"
#   else
#      error \
         "No 'mongocrypt-config.h' header is available. This must be generated in order to use libmongocrypt"
#   endif
#endif
```

Reply:
I'd like to enable that option as it really helps with readability, but I'd reserve that for a change where we re-apply formatting across the project(s), since that would have effects in many files.